### ISSUE TITLE
Reshape Layer

### ISSUE THREAD

**Issue by TyroD** [hollance/Forge#40]
Maybe i'm missing it in the documentation, but does Forge support a reshape layer?

**Comment by hollance** [comment]
Nope. In general, reshaping isn't necessary with Metal (although it depends on what you're trying to do). But even worse, it involves copying data so it's best avoided.
